### ISSUE TITLE
Motion to approve the Executive Council Decision Making Transparency

### ISSUE THREAD

**Issue by orchid00** [carpentries/executive-council-info#61]
I move to approve the decision making transparency rubric as follows: 

Transparency for how the Executive Council (EC) makes decisions. 

1. In order of relevance the EC should vote on:

- The strategic plan: approving mid to long term strategic direction priorities. Mid term 1 year and long term 3-5 years. Include voting on strategic alliances with outside organisations
- The yearly budget: oversee the yearly budget and annual financial reports.
- By-laws: modifying and approving official policy changes biannually
- Mission, vision, and values: defining, updating and approving the public mission, vision, and values statements
- Council-elected members: Policy to identifying, appointing, and removing/ replacing council-elected members when necessary
- Executive director: Appointing/removing the Executive Director

2. The EC should advise on but not vote on:

- Appointments of new staff: EC advice should continue to be considered alongside community members and The Core Team
- Equity, inclusion and access matters
- Approaches to growth in new regions
- Formation of committees/sub communities, specially if they affect multiple communities

3. The EC should be informed of (but no decision or advising expected):

- Decisions relating to Core Team changes
- The EC needs to be generally informed where groups have explicit powers that affect the culture of the organisation (e.g. Decisions of the Lessons Programs governance committees and Task Forces)

Notes:
This relates to Github issue: [#113](https://github.com/carpentries-ec/conversations_ec_ed/issues/113) on the  conversations_ec_ed/ repo
Text suggested based on the responses from EC and Core Team members (2020-2021)

**Comment by kariljordan** [comment]
Thanks @orchid00. Can you rephrase the motion such that it is clear what the EC will be voting on? For example,
you could say something like "I move to approve the decision making transparency rubric..."

We will then need a second, and the EC can vote. This is AWESOME!

**Comment by lexnederbragt** [comment]
Seconded

**Comment by smithmd5** [comment]
I second the motion.

**Comment by orchid00** [comment]
Thanks everyone, this motion is approved.

**Comment by orchid00** [comment]
will close this issue once this PR is merged https://github.com/carpentries/docs.carpentries.org/pull/732

**Comment by lexnederbragt** [comment]
PR merged, closing this issue.
